export/index.ts

```diff
@@ -358,11 +440,15 @@ export {
   NeonDbError,
 };
 
-export {
-  Connection,
-  DatabaseError,
-  Query,
-  ClientBase,
-  defaults,
-  types,
-} from 'pg';
+export type {
+  SocketDefaults,
+  FetchEndpointOptions,
+  WebSocketConstructor,
+  WebSocketLike,
+  subtls,
+} from '../shims/net';
+
+// @ts-ignore -- this is defined by esbuild so we can track whether we imported .js or .mjs
+const _bundleExt: 'js' | 'mjs' = BUNDLE_EXT;
```

Comment:
Can't we use `--out-extension:.js=.mjs` instead?

Reply:
Well, in `export.js` we already use `--outfile`.

These lines don't change what's exported, but they make it possible to tell, as a library consumer, which source file was imported/required by your code.

I added these in the course of debugging a problem recently. I had a theory that perhaps I was importing two separate versions of the library (CJS and ESM), which might be why the `neonConfig` parameters I was setting didn't seem to be getting used. Turned out I was wrong! But I think this is still a useful capability to have.

Reply:
Yy, it's probably fine, maybe let's just make it clearer on the comment that this is for debugging purposes then

Reply:
Done.